VERSION

```diff
@@ -1 +1 @@
-2.3.0
\ No newline at end of file
+3.1.0
```

Comment:
This file should not be committed. Please remove it and add it to the `.gitignore` file